.github/workflows/ci.yml

```diff
@@ -293,9 +293,7 @@ jobs:
     - name: Building Radare2
       run: |
         # treesitter doesnt build with this because of treesitter
-        export CFLAGS="-O2 -Werror -Wno-unused-result -std=gnu89 -Wno-unicode"
         export CFLAGS="-O2 -Werror -Wno-unused-result -Wno-unicode -Wno-unneeded-internal-declaration"
```

Comment:
i think just usin CFLAGS=-O2 is fine, that was the original behaviour and having those previous exports do nothing, the -Werror is making the build fail because of this harmless warning